torch/csrc/jit/runtime/static/te_wrapper.cpp

```diff
@@ -148,7 +148,7 @@ std::shared_ptr<TEWrapper> createRelu() {
   Tensor B = Compute("B", {N}, [&](const VarHandle& i) {
     auto zero = FloatImm::make(0.f);
     auto a = A.load(i);
-    return ifThenElse(a < zero, zero, a);
+    return CompareSelect::make(a, zero, zero, a, kLT);
```

Comment:
Wouldn't it be better to do `max(a, zero)` instead?

Reply:
Mm, maybe :) If that's so, we should update our standard lowering too:
https://github.com/pytorch/pytorch/blob/cf143bdc57826025f27017093e5d2a4d119b80d1/torch/csrc/jit/tensorexpr/lowerings.cpp#L586-L605

Reply:
There is a possibility that both cases end up with the same instructions eventually. It would be good to do a benchmark for this.